synapse/models/infotech.py

```diff
@@ -956,6 +966,32 @@ def getModelDefs(self):
                         'doc': 'The most recent version of the rule evaluated as a match.'}),
                 )),
 
+                ('it:reveng:function', {}, ()),
+
+                ('it:reveng:function:file', {}, (
+                    ('function', ('it:reveng:function', {}), {
+                        'ro': True,
+                        'doc': 'The hash matching the function'}),
+                    ('file', ('file:bytes', {}), {
+                        'ro': True,
+                        'doc': 'The file that contains the function.'}),
+                    ('va', ('int', {}), {
+                        'doc': 'The virtual address of the first codeblock of the function'}),
+                )),
+
+                ('it:reveng:function:string', {}, (
+                    ('function', ('it:reveng:function', {}), {
+                        'ro': True,
+                        'doc': 'The hash matching the function'}),
```

Comment:
"The GUID of the function which contains the string." or something :)

Reply:
XD :+1: